lib/pf/web/constants.pm

```diff
@@ -100,6 +100,7 @@ Readonly::Scalar our $REQ_ARUBA                 => 'apname';
 Readonly::Scalar our $REQ_CISCO_PORTAL          => '/cep(.*)';
 Readonly::Scalar our $REQ_RUCKUS                => 'sip';
 Readonly::Scalar our $REQ_AEROHIVE              => 'RADIUS-NAS-IP';
+Readonly::Scalar our $REQ_COOVACHILLI           => 'uamip';
```

Comment:
As discussed verbally, just change this to use the EXT_URL method and we're good to merge